resources/views/profile/edit.blade.php

```diff
@@ -76,8 +76,18 @@
 	<script src="{{mix('js/admin/profile/edit.js')}}"></script>
 
     <script>
+        var addons = [];
```

Comment:
I feel like this would be better in the layout blade.

Reply:
In principle I was trying to limit the scope of addons. I moved this code to the layout and base controller so that any page can have addons now.